.github/actions/release-please-semvar/action.yml

```diff
@@ -32,3 +32,40 @@ runs:
         git push origin v${{ steps.release.outputs.major }}
         git push origin v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}
       if: ${{ steps.release.outputs.release_created }}
+    - name: tag component major and minor versions
+      if: ${{ steps.release.outputs.release_created }}
+      uses: actions/github-script@v7
+      env:
+        release_output: ${{ toJSON(steps.release.outputs) }}
+      with:
+        result-encoding: string
+        script: |
+          exec.exec('git config user.name czi-github-helper[bot]')
+          exec.exec('git config user.email 95879977+czi-github-helper[bot]@users.noreply.github.com')
+          
+          const outputs = JSON.parse(process.env.release_output);
```

Comment:
@jakeyheath I found the pattern of using the the environment to be helpful in overcoming the problem of escaping large variables in templated scripts

Reply:
sick!